### DISCUSSION TITLE
Améliorer le compte Admin

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1184] [Idées]
Une suggestion pour le compte Admin : sur la fiche d'une cantine, pouvoir cliquer sur le nom du gestionnaire pour atterrir directement sur la fiche du gestionnaire.

Intérêt : gain de temps.

Exemple d'utilisation : lorsque je consulte la cantine d'un gestionnaire qui gère plusieurs cantines et que je veux voir toutes les cantines qu'il gère, plutôt que d'aller ensuite dans Utilisateurs, puis de rechercher son nom, puis de cliquer sur son profil, je n'aurais qu'à cliquer sur son nom directement sur la fiche de la cantine.

<img width="1440" alt="Capture d’écran 2022-03-16 à 09 15 52" src="https://user-images.githubusercontent.com/93709421/158547405-651c53f6-e4cf-4fac-bd0c-436f28092024.png">
<img width="1440" alt="Capture d’écran 2022-03-16 à 09 20 17" src="https://user-images.githubusercontent.com/93709421/158547414-bb034a43-404f-420a-8385-d642d77705c3.png">

**Comment by hfroot** [comment]
Merci pour la suggestion. C'est difficile de faire ça, car pour enlever/ajouter un.e gestionnaire dans l'endroit que tu propose, on doit cliquer sur le nom déjà.
